pymeasure/instruments/anritsu/anritsuMS464xB.py

```diff
@@ -0,0 +1,844 @@
+        values are between 0 and 255. Refer to the instrument manual for an explanation of the bits.
+        """,
+        values=[0, 255],
+        validator=strict_range,
+        cast=int,
+    )
+
+    def query_event_status_register(self):
+        """ Query the value of the Standard Event Status Register.
+
+        Note that querying this value, clears the register. Refer to the instrument manual for an
+        explanation of the returned value.
+        """
+        return self.values("*ESR?", cast=int)[0]
+
+    service_request_enable_bits = Instrument.control(
+        "*SRE?", "*SRE %d",
+        """Control the Service Request Enable Register bits.
+
+        Valid values are between 0 and 255; setting 0 performs a register reset. Refer to the
+        instrument manual for an explanation of the bits.
+        """,
+        values=[0, 255],
+        validator=strict_range,
+        cast=int,
+    )
+
+    def return_to_local(self):
+        """ Returns the instrument to local operation. """
+        self.write("RTL")
+
+    binary_data_byte_order = Instrument.control(
+        ":FORM:BORD?", ":FORM:BORD %s",
+        """Control the binary numeric I/O data byte order.
+
+        valid values are:
+
+        =====   =========================================
+        value   description
+        =====   =========================================
+        NORM    The most significant byte (MSB) is first
+        SWAP    The least significant byte (LSB) is first
+        =====   =========================================
+        """,
+        values=["NORM", "SWAP"],
+        validator=strict_discrete_set,
+    )
+
+    max_number_of_points = Instrument.control(
+        ":SYST:POIN:MAX?", ":SYST:POIN:MAX %d",
+        """Control the maximum number of points the instrument can measure in a sweep.
+
+        Note that when this value is changed, the instrument will be rebooted.
+        Valid values are 25000 and 100000. When 25000 points is selected, the instrument supports 16
+        channels with 16 traces each; when 100000 is selected, the instrument supports 1 channel
+        with 16 traces.
+        """,
+        values=[25000, 100000],
+        validator=strict_discrete_set,
+        cast=int,
+    )
+
+    number_of_channels = Instrument.control(
+        ":DISP:COUN?", ":DISP:COUN %d",
+        """Control the number of displayed (and therefore accessible) channels.
+
+        When the system is in 25000 points mode, the number of channels can be 1, 2, 3, 4, 6, 8, 9,
+        10, 12, or 16; when the system is in 100000 points mode, the system only supports 1 channel.
+        If a value is provided that is not valid in the present mode, the instrument is set to the
+        next higher channel number.
+        """,
+        values=[1, 16],
+        validator=strict_range,
+        cast=int,
+    )
+
+    display_layout = Channel.control(
```

Comment:
Should be `Instrument.control`, right?

Reply:
They are actually the same. For future compatibility (if they might diverge), it is better to use the appropriate prefix.

Reply:
That's a good thing to check that I didn't think of yet; thanks